mappings/net/minecraft/server/world/ChunkTicketManager.mapping

```diff
@@ -67,8 +67,10 @@ CLASS net/minecraft/class_3204 net/minecraft/server/world/ChunkTicketManager
 		ARG 2 pos
 		ARG 3 level
 		ARG 4 argument
+	METHOD method_20800 hasTicket (J)Z
+		ARG 1 posLong
 	METHOD method_21623 getTicket (J)Ljava/lang/String;
-		ARG 1 pos
+		ARG 1 posLong
```

Comment:
`posLong` is not more specific than `pos` given the type is already `J`. Maybe `chunkPos` is better

Reply:
I kind of wanted to be specific that it's a ChunkPos that's been parsed/packed up into a long. It's not really the same as just a long, nor the same ChunkPos itself as the result of `Object.hashCode()` isn't necessarily the same as the object itself.

Reply:
Well, chunk positions have unique bidirectional mappings from and to a long, while hashCode operation is not bidirectional. So this `pos` is a pos, just in long format for convenience. Maybe `chunkPos`, but `posLong` is not more helpful.